rclcpp/test/rclcpp/waitables/waitable_test_helpers.hpp

```diff
@@ -0,0 +1,117 @@
+// Copyright 2024 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLCPP__WAITABLES__WAITABLE_TEST_HELPERS_HPP_
```

Comment:
#pragma once

Reply:
We don't use pragma once.

Reply:
hu, why ?

Reply:
Technically, `#pragma once` is not part of the standard.

Reply:
^ that.

We've discussed changing them to random strings for a few reasons, but not to change to pragma once.